cookbooks/bcpc-hadoop/libraries/utils.rb

```diff
@@ -439,3 +439,22 @@ def update_oozie_sharelib(host)
     Chef::Log.info("Sharelibupdate: Oozie server not running on #{host}")
   end
 end
+
+def get_cluster_nodes()
+
+  cluster_file = node['bcpc']['cluster']['file_path']
```

Comment:
@amithkanand Could you provide a comment here that this must be run on a machine with `cluster.txt` to avoid someone errantly trying to use this function?